lighthouse-core/report/html/renderer/util.js

```diff
@@ -288,12 +288,16 @@ class Util {
    * @return {{file: string, hostname: string, origin: string}}
    */
   static parseURL(url) {
-    const parsedUrl = new URL(url);
-    return {
-      file: Util.getURLDisplayName(parsedUrl),
-      hostname: parsedUrl.hostname,
-      origin: parsedUrl.origin,
-    };
+    try {
```

Comment:
why was this needed?

Reply:
well we only *need* this or a fix on the `_renderTextURL` to be looser on its try/catch (we check instanceof TypeError), but it seemed like a good catch on passing invalid URLs to `parseURL`. We pass through several `'Unknown'` strings to this function which throw

Reply:
hmm, but why was it needed for this PR? If it's throwing shouldn't it have caused a failure before, too? Or is the problem something else.

Reply:
well again it isn't strictly *needed*, but seemed like a good idea anyhow.

full details:
in jest's jsdom env, the URL class throws an error that isn't instanceof TypeError
`_renderTextURL` which normally catches TypeErrors thrown by this method was then throwing

this isn't needed anymore because along the way I turned off default jsdom environment, but it seemed like a good idea to keep, it seems like you don't think so though so I'll revert :)